README.md

```diff
@@ -1,6 +1,6 @@
-# dojo-php
+# Dojo PHP SDK
 
-The official [Dojo][dojo] PHP library, supporting Remote Payments API.
+The official [Dojo][dojo] PHP library, supporting [Remote Payments API][api-docs].
```

Comment:
```suggestion
The official [Dojo][dojo] PHP library, supporting the [Payments API][api-docs].
```

Reply:
Not sure what we're calling this API going forward, but it's Payments on all the docs for now

Reply:
Right. Fixed. Thanks!